.github/workflows/build.yml

```diff
@@ -28,33 +22,35 @@ jobs:
       - name: Update Fastlane
         run: bundle update fastlane
 
+        # Required for KotlinMultiplatform
+      - uses: actions/setup-java@v1
+        with:
+          java-version: '11'
+
       - name: Pod Install
         run: bundle exec fastlane cru_shared_lane_pod_install
```

Comment:
any reason you are preferring to install the pods via fastlane instead of just running `pod install` here?

Reply:
Hey @frett no preferred reason here, I hadn't thought about using pod install, but will try that, one less lane to have.

Reply:
Using more direct steps when fastlane isn't actually required (e.g. cocoapods) makes it much more understandable for users who don't know fastlane.